### PR TITLE
closes #25

### LINKED ISSUE #25: Create User Model

### PR DESCRIPTION
**Pull request by gabrielwr**
closes user model #25

### CODE REVIEW

File: db/models/user.js [hunk 1]
```diff
@@ -2,18 +2,42 @@
...
+    type: STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    }
+  },
+  user_name: {
+    type: STRING,
+    allowNull: false
+  },
```
Comment (JeffKandel) [inline db/models/user.js]: Let's add not Empty validation here too!

File: db/models/user.js [hunk 2]
```diff
@@ -2,18 +2,42 @@
...
   email: {
```
Comment (JeffKandel) [inline db/models/user.js]: Let's make this unique! :-D

File: db/models/user.js [hunk 3]
```diff
@@ -2,18 +2,42 @@
...
     type: STRING,
+    allowNull: false,
     validate: {
       isEmail: true,
       notEmpty: true,
     }
   },
+  is_admin: {
+    type: BOOLEAN
```
Comment (JeffKandel) [inline db/models/user.js]: Let's set the default Value to false

**Review by JeffKandel — CHANGES_REQUESTED** [review]
See changes below.

**Review by lada-tech — APPROVED** [review]
all good

**Review by JeffKandel — APPROVED** [review]
This is done!

**Pull request merged**